plugin/collector/snap-plugin-collector-mock2/mock/mock.go

```diff
@@ -81,6 +81,15 @@ func (f *Mock) CollectMetrics(mts []plugin.MetricType) ([]plugin.MetricType, err
 			panic("Oops!")
 		}
 
+		if c, ok := mts[i].Config().Table()["time_to_wait"]; ok {
+			dur, err := time.ParseDuration(c.(ctypes.ConfigValueStr).Value)
+			if err != nil {
+				log.Println(err.Error())
+			} else {
+				time.Sleep(dur)
+			}
+		}
+
```

Comment:
@IzabellaRaulin thanks for testing it. I agree with you - I don't think it's necessary to land this change on master. I added another test in scheduler_test.go which has timeToWait in collect method and an assertion on the total elapsed time. In my opinion, it would suffice to land this test on master.